src/bridge/cockpitpolkithelper.c

```diff
@@ -144,7 +145,8 @@ main (int argc, char *argv[])
 
       if (ferror (stdout))
         {
-          warnx ("couldn't write to stdout");
+          if (errno != EPIPE)
```

Comment:
Stef Walter <notifications@github.com> writes:

>> @@ -144,7 +145,8 @@ main (int argc, char *argv[])
>>
>>        if (ferror (stdout))
>>          {
>> -          warnx ("couldn't write to stdout");
>> +          if (errno != EPIPE)
>
> How do we get the errno from a FILE error?

I don't know of any way.  I think we have to check errno immediately
after each operation that might fail.